src/tests/test_index_btree.cpp

```diff
@@ -501,7 +501,7 @@ struct BtreeConcurrentTest : public BtreeTestHelper< TestType >, public ::testin
     }
 
 private:
-    const std::string m_shadow_filename = "shadow_map.txt";
+    const std::string m_shadow_filename = "/tmp/shadow_map.txt";
```

Comment:
This will conflict for different users using same build machine. Can you keep it same and delete it after UT runs ?

Reply:
This file will be deleted after the last run of each type` function_normal(0, True, False, type) # cleanup after shutdown`